network_modeling/configs/templates/dellmellanox/tds/leaf-bmc-to-spine.lag.j2

```diff
@@ -0,0 +1,26 @@
+{#- leaf-bmc-to-spine #}
+{%- set ns = namespace(lag_displayed=false) %}
+{%- for node in cabling.nodes if node.subtype == "spine" %}
+{%- if ns.lag_displayed == false %}
+{%- set ns.lag_displayed = true %}
+interface lag 255
```

Comment:
should look similar to this.

https://stash.us.cray.com/projects/CASMNET/repos/switch-configs/browse/shandy-sw-leaf-001#127-133,634-652